salt/_modules/caasp_version.py

```diff
@@ -0,0 +1,18 @@
+"""This module provide version information to be used (mostly) in Jinja
+templates.
+"""
+UNKNOWN_VERSION = (0, 0)
+
+
+def __virtual__():
+    return "caasp_version"
+
+
+def caasp_version():
+    # Python 3 does not allow comparing list & tuples, so force a tuple:
+    version = tuple(__salt__['grains.get']('osrelease_info', UNKNOWN_VERSION))
+    return version
+
+
+def use_registry_images():
+    return __salt__['grains.get']('use_registry_images', False)
```

Comment:
Same comment on this one, if some machines will allow to `use_registry_images`, then it makes sense to have it as a grain, otherwise (if it's cluster-wide) I'd say having this information inside the pillar would be better.